lib/rubocop/cop/naming/constant_name.rb

```diff
@@ -8,6 +8,15 @@ module Naming
       #
       # To avoid false positives, it ignores cases in which we cannot know
       # for certain the type of value that would be assigned to a constant.
+      #
+      # @example
+      #   # bad
+      #   InchInCm = 2.54
+      #   INCHinCM = 2.54
+      #   Inch_In_Cm = 2.54
```

Comment:
I'm not sure if these examples are any good. Anything that starts with a capital letter is considered a constant, so any variation of that would be a candidate example of a bad constant name.

Reply:
They are fine.